### PR TITLE
fix: argo submit management of subdirectories

### PR DESCRIPTION
**Pull request by MDavidson17**
Subdirectly management didn't actually work correctly

### CODE REVIEW

File: .github/workflows/publish-to-odr.yml [hunk 1]
```diff
@@ -61,13 +61,13 @@ jobs:
         id: modified-files
         run: |
           # AM = Include: Added, Modified
-          echo "modified_files=$(git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.event.after }})" >> $GITHUB_OUTPUT
+
+          MODIFIED_FILEs=$(git diff --name-only --diff-filter=AM ${{ github.event.before }} ${{ github.event.after }})
```
Comment (l0b0) [inline .github/workflows/publish-to-odr.yml:65]: This isn't going to work - `MODIFIED_FILEs` != `MODIFIED_FILES`.

File: .github/workflows/publish-to-odr.yml [hunk 2]
```diff
@@ -61,13 +61,13 @@ jobs:
...
+          MODIFIED_PARAMETERS_FILES=$(echo $MODIFIED_FILES | grep "publish-odr-parameters/" | grep ".yaml\$" | sort | uniq)
+          echo "modified_files=${MODIFIED_PARAMETERS_FILES}" >> $GITHUB_OUTPUT
```
Comment (l0b0) [inline .github/workflows/publish-to-odr.yml:67]: This value is only used in the subsequent step, so is it actually necessary to use `GITHUB_OUTPUT`?

File: .github/workflows/publish-to-odr.yml (hunk 1 above)
Comment (MDavidson17) [reply .github/workflows/publish-to-odr.yml]: :eyes: good spot!

File: .github/workflows/publish-to-odr.yml (hunk 2 above)
Comment (l0b0) [reply .github/workflows/publish-to-odr.yml:67]: Yep. Basically, adding `GITHUB_OUTPUT` to the mix complicates things a fair bit, without adding any particular value other than splitting the collecting of paths from their use. It's a trade-off, but I think that means we should probably put them together.

**Review by l0b0 — APPROVED** [review]
LGTM. As discussed, we probably want to test this with both a modified and a new file.

**Pull request merged**